c_src/cmake/FindErlang.cmake

```diff
@@ -0,0 +1,144 @@
+# Distributed under the OSI-approved BSD 3-Clause License.  See accompanying
+# file Copyright.txt or https://cmake.org/licensing for details.
+
+#[=======================================================================[.rst:
+FindErlang
+-------
+
+Finds Erlang libraries.
+
+Imported Targets
+^^^^^^^^^^^^^^^^
+
+This module provides the following imported targets, if found:
+
+``Erlang::Erlang``
+  Header only interface library suitible for compiling NIFs.
+
+``Erlang::EI``
+  Erlang interface library.
+
+``Erlang::ERTS``
+  Erlang runtime system library.
+
+Result Variables
+^^^^^^^^^^^^^^^^
+
+This will define the following variables:
+
+``Erlang_FOUND``
+  True if the system has the Erlang library.
+``Erlang_RUNTIME``
+  The path to the Erlang runtime.
+``Erlang_COMPILE``
+  The path to the Erlang compiler.
+``Erlang_EI_PATH``
+  The path to the Erlang erl_interface path.
+``Erlang_ERTS_PATH``
+  The path to the Erlang erts path.
+``Erlang_EI_INCLUDE_DIRS``
+  /include appended to Erlang_EI_PATH.
+``Erlang_EI_LIBRARY_PATH``
+  /lib appended to Erlang_EI_PATH.
+``Erlang_ERTS_INCLUDE_DIRS``
+  /include appended to Erlang_ERTS_PATH.
+``Erlang_ERTS_LIBRARY_PATH``
+  /lib appended to Erlang_ERTS_PATH.
+
+#]=======================================================================]
+include(FindPackageHandleStandardArgs)
+
+SET(Erlang_BIN_PATH
+  $ENV{ERLANG_HOME}/bin
+  /opt/bin
+  /sw/bin
+  /usr/bin
+  /usr/local/bin
+  /opt/local/bin
+  )
```

Comment:
Can you expand on that? You may have a valid point that I'm not aware of.

Reply:
No, I don't think I have a valid point since NIF binaries probably never get installed in `/usr/sbin`.

In the past, I've written `.mk` files that install escripts into `/usr/sbin`. These escripts were mostly tests for manufacturing. The ECC had one as did GPS when that was SPI. In retrospect, I'm not sure that `/usr/sbin` was the right place to put them since technically they're not binaries (they're Erlang) and they don't necessarily require root privileges (the "s" in "sbin" stands for superuser).